assets/js/app.js

```diff
@@ -76,4 +76,36 @@ app.addComponent('http', new httpComponent(app));
 app.run();
 
 //Access for inline scripts
-window.app = app;
\ No newline at end of file
+window.app = app;
+
+//Form validation
+
+function validateForm(formName) {
```

Comment:
One more improvement can be done. Add it as a module. You can simple look at the ajax/http component.

Ask me if you need there help

Reply:
So I sperate validateForm from app.js and create a new component?

Reply:
Did quite get what you meant. The new commit has the valForm function in a seperat js file and works now. But doesn't get 'build' by yarn.